build-scripts/build-auth-rpm

```diff
@@ -170,7 +170,8 @@ make install DESTDIR=%{buildroot}
 
 %{__rm} -f %{buildroot}%{_libdir}/%{name}/*.la
 %{__install} -p -D -m 0755 pdns/pdns %{buildroot}%{_initrddir}/pdns
-%{__mv} %{buildroot}%{_sysconfdir}/%{name}/pdns.conf{-dist,}
+%{buildroot}/usr/bin/pdns_server --config > %{buildroot}%{_sysconfdir}/%{name}/pdns.conf
+%{__rm} %{buildroot}%{_sysconfdir}/%{name}/pdns.conf-dist
```

Comment:
also, recommend putting `--no-config` in front of `--config` to avoid (launch) contamination - I believe this issue is theoretical in the current startup flow but that could change.